pages/series/index.vue

```diff
@@ -91,7 +91,7 @@ onBeforeMount(async () => {
   line-height: 28px;
   align-self: end;
   text-decoration: none;
-  color: var(--text-color);
+  color: var(--text-color) !important;
```

Comment:
I also had to add `.projects-landing-page__project a:visited` to the link styles because the visited links were blue - not sure if that's still happening

Reply:
hmmm... I didn't see the visited issue after I added the !important... but I'll add the additional selector in the css anyway.